### ISSUE TITLE
[BUG] After First launch, the prompt txt box doesn't auto adjust its size to fit the prompt length, in both t2i and i2i

### ISSUE THREAD

**Issue by LiJT** [hlky/stable-diffusion-webui#301]
**Describe the bug**
After the first time webui launched, the prompt txt box doesn't auto adjust its size to fit the prompt length, until you try to click Submit on enter yes or no button, then the prompt txt box will start to response. I think this is the tiny bug.

**To Reproduce**
1. run webui.cmd
2. go to http://localhost:7860/
3. enter any text in the prompt box, no matter how long the txt is, the box won't resize. 

**Expected behavior**
The prompt box should be able to resize at any time, in both text2image and image2image. 
Currently they both not working initially. But t2t can recover itself after random button clicked.
In text2image tab, the txt box will recover itself after random button clicked.
But in image2image tab, the box wont resize at all

**Screenshots**
![t2i_1](https://user-images.githubusercontent.com/11270197/187352390-10aafb0b-ba01-4258-82ce-8d5eea35f521.png)
Initially its not working properly

![t2i_2](https://user-images.githubusercontent.com/11270197/187352413-6ff186d1-854e-4c74-a7ca-ea16c27075ee.png)
After I click this button, the txt box respond and refresh itself back to normal

![i2i](https://user-images.githubusercontent.com/11270197/187352566-ff997cd1-14cc-43a3-8f82-d9698c691044.png)
i2i on the other hand not working at all

**Comment by hlky** [comment]
Extremely minor UI bug that doesn't particularly need attention right now because entering 1 prompt per line isn't actually implemented yet, the multi line prompt thing is actually just used by people who separate the components of their prompt onto separate lines so they can change it easier.

**Comment by hlky** [comment]
Ok this should have been fixed with #304 

you'll need to wait for the changes to be synced to main repo unless you understand how to update from dev repo -> main repo yourself

```
Dev repo is set up to be easy for developers so the structure is different.
Changes are synced to main repo.
GitHub Actions puts things in the right places.
If you can't tell where things go check .github/sync.yml
```
